test/functional/android/context_switching_tests.py

```diff
@@ -23,43 +21,39 @@
 
 
 @pytest.mark.skip(reason="Need to fix broken test")
-class ContextSwitchingTests(unittest.TestCase):
-    def setUp(self) -> None:
+class TestContextSwitching(object):
+    def setup_method(self, method) -> None:  # type: ignore
```

Comment:
Need to check if `method` can be removed.

Reply:
https://docs.pytest.org/en/latest/xunit_setup.html#method-and-function-level-setup-teardown

`As of pytest-3.0, the method parameter is optional.`